scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -218,7 +218,7 @@ public boolean kill() {
             try {
                 infrastructure.getRMProxiesManager().getUserRMProxy(taskData.getUser(),
                         taskData.getCredentials())
-                        .releaseNodes(nodes, taskData.getTask().getCleaningScript(), null);
+                        .releaseNodes(nodes, taskData.getTask().getCleaningScript(), null,null,null);
```

Comment:
Is it correct to pass null null null? You think we can have like 2 separate methods or to wrap those nulls in a single object? when it's more than 3 parameters in a method it's normally difficult to use and understand.